src/views/pages/students.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import StudentStore from '../../store/student-store';
+import Student from '../components/student.jsx';
+import StudentPageController from '../../controllers/pages/students';
+
+export default class Models extends ListenerComponent {
+
+    componentWillMount(){
+        StudentPageController.getStudents();
+    }
+
+    getStores() {
+        return [
+            StudentStore
+        ];
+    }
+
+    getState() {
+        return {
+            students: StudentStore.getAllStudents()
+        };
+    }
+
+    render() {
+        return (
+            <div id="students">
+                <h1>All Students</h1>
+                {this.state.students.sort(this.studentSort).map((student) => {
+                    return (
+                        <div key={student.id}>
+                            <Student student={student}  />
+                        </div>
+                    );
+                })}
+            </div>
+        );
+    }
+
+    studentSort(a, b){
+        let splitA, splitB, lastA, lastB;
+        splitA = a.name.split();
+        lastA = splitA[splitA.length - 1];
+        splitB = b.name.split();
+        lastB = splitB[splitB.length - 1];
+        return lastA.localeCompare(lastB);
+    }
```

Comment:
Also, do we know if students will always have a middle name/initial? The example function I provided would sort them by last name, then by middle name/initial, and then by the first name if they have all three. If one only has a first and last name, their first name would be sorted against other students' middle names. And this doesn't account for multi-part surnames such as "von Neumann"...

For example, this would convert "John von Neumann" to "Neumann von John" for sorting when it should probably treat it as "von Neumann John". It would also compare "Alice J Smith" as "Smith J Alice" and "Bob Smith" as "Smith Bob", so Bob Smith would be sorted before Alice Smith. Would we rather sort by last name, then first name, then middle?

It gets kind of messy... Maybe the simple split/reverse/join approach is good enough.